skypy/pipeline/tests/test_driver.py

```diff
@@ -90,29 +82,40 @@ def test_driver():
     # Check variables intialised by value
     config = {'test_int': 1,
               'test_float': 1.0,
-              'test_string': 'hello world'}
+              'test_string': 'hello world',
+              'test_list': [0, 'one', 2.],
+              'test_dict': {'a': 'b'}}
     driver = SkyPyDriver()
     driver.execute(config)
     assert isinstance(driver.test_int, int)
     assert isinstance(driver.test_float, float)
     assert isinstance(driver.test_string, str)
+    assert isinstance(driver.test_list, list)
+    assert isinstance(driver.test_dict, dict)
     assert driver.test_int == 1
     assert driver.test_float == 1.0
     assert driver.test_string == 'hello world'
+    assert driver.test_list == [0, 'one', 2.]
+    assert driver.test_dict == {'a': 'b'}
 
     # Check variables intialised by function
-    config = {'test_func': {
-                'function': 'list',
-                'args': [
-                    '"hello world"']},
-              'test_func2': {
-                'function': 'len',
-                'args': [
-                    'test_func']}}
+    config = {'test_func': ('list', 'hello world'),
+              'test_func2': ('len', 'test_func')}
```

Comment:
```suggestion
              'test_func2': ('len', ('test_func',))}
```
This is supposed to be a reference to the variable `test_func` not the literal string `"test_func"`. I think this is the correct new syntax for that. Hence the test below should remains as `assert driver.test_func2 == 11`

Reply:
Hah I couldn't figure out why it was 11 🤡